dashboard/config/scripts/levels/2-3 Artist 1 new.level

```diff
@@ -38,7 +38,19 @@
     "callout_json": "[]"
   },
   "published": true,
-  "notes": ""
+  "notes": "",
+  "level_concept_difficulty": {
+    "sequencing": 3,
+    "debugging": null,
+    "repeat_loops": null,
+    "repeat_until_while": null,
+    "for_loops": null,
+    "events": null,
+    "variables": null,
+    "functions": null,
+    "functions_with_params": null,
+    "conditionals": null
```

Comment:
Probably, this is just the way `Level.serializable_hash(:include => :level_concept_difficulty)` works by default. I'll see if it (and more importantly `assign_nested_attributes_for_one_to_one_association`) are happy with leaving out nulls